src/vtk_calculator.py

```diff
@@ -44,25 +44,48 @@ def parse_args():
                         help="The mesh (VTK Unstructured Grid) used as input")
     parser.add_argument("--function", "-f", dest="function", required=True,
                         help="""The function to evalutate on the mesh.
-            Syntax is the same as used in the calculator object, coordinates are given as e.g.  'cos(x)+y'.""")
+            Syntax is the same as used in the calculator object, coordinates are given as e.g.  'cos(x)+y'.
+            Alternatively, you can use predefined function visit preCICE documentation.""")
```

Comment:
```suggestion
            The syntax is the same as used in the calculator object, where coordinates are given by 'xyz' e.g.  'cos(x)+y'.
            Alternatively, you can use predefined functions such as the Eggholder or Franke's function. Please visit the preCICE documentation for a complete list of available functions.""")
```

Reply:
Referring to the "print the functions"-option would be useful here:

> Please use --functions to display all options and visit the preCICE documentation for the complete reference.

Reply:
I changed to this way it prints all they keys in functions dictionary, thank you.